cdap-watchdog/src/main/java/co/cask/cdap/metrics/query/MetricsHandler.java

```diff
@@ -168,6 +170,31 @@ public void query(HttpRequest request, HttpResponder responder,
     }
   }
 
+  @POST
```

Comment:
Can we use MetricStore interface to delete metrics on delete of namespace? We don't want to expose REST API just yet